services/manufacturerProdRangeService.js

```diff
@@ -0,0 +1,65 @@
+"use strict"
+
+const ManufacturerProdRange = require(`${__dirname}/../models/manufacturerProdRangeModel`)
+const { errObject } = require(`${__dirname}/../helpers/helper`)
+
+exports.getPagedManufacturerProdRanges = async (pageNo, docsPerPage) => {
+	try {
+		const totalDocs = await ManufacturerProdRange.countDocuments()
+		const genericPipeData = await ManufacturerProdRange.find()
+			.populate("manufacturer")
+			.populate("productType")
+			.populate("materialType")
+			.populate("connectionType")
+			.populate("pipeSizesAvailable")
+			.skip(pageNo * docsPerPage)
+			.limit(docsPerPage)
+
+		const response = {
+			total: totalDocs,
+			genericPipeData,
+		}
+
+		return response
+	} catch (error) {
```

Comment:
this try catch redundant and not needed.
check all try catches.

Reply:
done